style.css

```diff
@@ -909,6 +909,14 @@ a:hover .nav-title,
 	color: #fff;
 }
 
+/* Fixes linked images */
+.entry-content a img,
+.widget a img {
+	background: #fff;
+	box-shadow: 0 0 0 6px #fff;
+	filter: progid:DXImageTransform.Microsoft.DropShadow(OffX=0, OffY=5, Color=#ffffff);
```

Comment:
What do you think about moving this to the ie8.css stylesheet, since it's only needed for that version of IE?